### DISCUSSION TITLE
Updating Jellyfin

### DISCUSSION THREAD

**Discussion by EdorasRP-system** [community-scripts/ProxmoxVE#433] [Q&A] [answered]
Hello!

Just a quick question, using `update` in the LXC for Jellyfin is the only way to update Jellyfin, do I have to wait for the update from this repository? I see the server version is 10.9.11 and the web version is on 10.10.3 I keep getting a message on my client ap to update my server 😅

**Comment by tesla-76** [comment]
The scrip normally always applies in order to upgrade to the latest version of a program released on the respective repository.

**Comment by EdorasRP-system** [comment]
I see, for some reason I can't update to the last version, and no errors on the update
<img width="246" alt="brave_gJFfagqqG4" src="https://github.com/user-attachments/assets/7132ee8b-e34f-4eac-8db6-cc0050fb58e5">

**Reply by Arnaud1977** [reply]
same here.

**Reply by Vakrehus** [reply, marked as answer]
```
apt update
apt full-upgrade -y
```

**Comment by ceeboonan** [comment]
Hello!
I did `apt-get update` but got kept back packages for jellyfin and jellyfin-web because of potential dependency issues. To resolve this issue i did `apt-get dist-upgrade` and both are on 10.10.3 now.

Happy updating!

**Comment by jihemsey** [comment]
"apt-get update"
"apt-get --with-new-pkgs upgrade jellyfin jellyfin-server" 
great to update from 10.10.1 to 10.10.2 when "kept back packages"
After the script "update" works.
For me.

**Comment by johndoe0815** [comment]
What is then 
`bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/jellyfin.sh)"`
doing?
It says that it would update Jellyfin:
![grafik](https://github.com/user-attachments/assets/ad177c11-8b30-4d58-bd3b-c237ef86a2f1)
It also seems to do so according to the displayed version number in Jellyfin (at least web and build version), however the update process spins endlessly (I waited several hours and it still did not say that is finished).

apt-get is not even mentioned on the overview page:
https://community-scripts.github.io/ProxmoxVE/scripts?id=jellyfin

**Reply by GilDev** [reply]
[Here’s what it does](https://github.com/community-scripts/ProxmoxVE/blob/main/ct/jellyfin.sh#L56-L67)

**Reply by johndoe0815** [reply]
Shouldn’t then be the script enough? And if not, why are additional necessary commands not added to the script? I am still confused, which commands to use to reliably update Jellyfin.

**Comment by jcisio** [comment]
This PR should fix it https://github.com/community-scripts/ProxmoxVE/pull/1771
